src/index.css

```
@@ -1,41 +1,81 @@
+/* --- General Reset and Box-Sizing --- */
+
+/* Ensure consistent box-sizing across all elements */
+*, *::before, *::after { box-sizing: border-box; /* Prevents padding and borders from affecting element size */ }
+
+/* Reset margin and padding for all elements */
+* {
+	margin: 0;
+	padding: 0; /* Removes default spacing */
+}
+
+/* --- Remove Default Styling for Lists --- */
+ul, ol { list-style: none; /* Removes default bullet points and numbering */ }
+
+/* --- Remove Default Button Styles --- */
+button {
+	all: unset; /* Removes all default styles applied by the browser */
+	cursor: pointer; /* Ensures buttons have a clickable cursor */
+}
+
+/* --- Ensure Images are Responsive --- */
+img {
```

```suggestion
:where(img, iframe, svg) {
```

We want to apply that to all "inline-ish" elements, in that case, those are images, iframes and inline SVGs[*].

The [`:where()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:where) selector here helps as it keeps the [selector specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_cascade/Specificity) at 0, so every rule will overwrite it. This helps avoid unintended consequences like the use of `!important`.

[*]: We may carve out an exception for SVGs here like icons, where they have different styles applied.